pkg/engine/engine.go

```diff
@@ -51,6 +53,7 @@ func NewEngine(
 	ivCache imageverifycache.Client,
 	contextLoader engineapi.ContextLoaderFactory,
 	exceptionSelector engineapi.PolicyExceptionSelector,
```

Comment:
This is important for the CLI given that it now supports exceptions. But, I think we can remove it and create a `store` in the CLI for exceptions. 

Currently, we are [creating an exception lister](https://github.com/kyverno/kyverno/blob/07a6bf42f52e087f409d26bb4e16f682c0f9e2dd/cmd/cli/kubectl-kyverno/processor/policy_processor.go#L64-L66) in the CLI and passing it to the [NewEngine()](https://github.com/kyverno/kyverno/blob/main/cmd/cli/kubectl-kyverno/processor/policy_processor.go#L83C3-L83C24). Instead, a `store` can be initialized and then it can be used in the engine. WDYT?

Reply:
Implemented for the CLI and removed the Lister